client/src/www/app/outline_server_repository/index.ts

```diff
@@ -120,11 +119,7 @@ export class OutlineServerRepository implements ServerRepository {
     if (alreadyAddedServer) {
       throw new errors.ServerAlreadyAdded(alreadyAddedServer);
     }
-    config.validateAccessKey(accessKey);
-
-    // Note that serverNameFromAccessKey depends on the fact that the Access Key is a URL.
-    const serverName = config.serviceNameFromAccessKey(accessKey);
-    const server = this.createServer(uuidv4(), accessKey, serverName);
+    const server = this.createServer(uuidv4(), accessKey, undefined);
```

Comment:
I think we can ignore the last optional parameter?

```suggestion
    const server = this.createServer(uuidv4(), accessKey);
```

Reply:
I'd rather make the parameters explicit. This makes it clear there's no previous name, even though it may have.